src/Debugging/Microsoft.DiaSymReader.PortablePdb/Microsoft.DiaSymReader.PortablePdb.nuspec

```diff
@@ -13,22 +13,22 @@
         <dependency id="System.Collections.Immutable" version="1.1.37" />
       </group>
       <group targetFramework="netstandard1.1">
-        <dependency id="Microsoft.DiaSymReader" version="1.0.8-rc2-60608-04" />
-        <dependency id="System.Collections" version="4.0.11-rc3-24208-04" />
-        <dependency id="System.Collections.Immutable" version="1.2.0-rc3-24208-04" />
-        <dependency id="System.Diagnostics.Debug" version="4.0.11-rc3-24208-04" />
-        <dependency id="System.Globalization" version="4.0.0-rc3-24208-04" />
-        <dependency id="System.IO" version="4.1.0-rc3-24208-04" />
-        <dependency id="System.Linq" version="4.1.0-rc3-24208-04" />
-        <dependency id="System.Reflection" version="4.1.0-rc3-24208-04" />
-        <dependency id="System.Reflection.Metadata" version="1.3.0-rc3-24208-04" />
-        <dependency id="System.Reflection.Primitives" version="4.0.1-rc3-24208-04" />
-        <dependency id="System.Runtime" version="4.1.0-rc3-24208-04" />
-        <dependency id="System.Runtime.Extensions" version="4.1.0-rc3-24208-04" />
-        <dependency id="System.Runtime.InteropServices" version="4.1.0-rc3-24208-04" />
-        <dependency id="System.Text.Encoding" version="4.0.11-rc3-24208-04" />
-        <dependency id="System.Threading" version="4.0.11-rc3-24208-04" />
-        <dependency id="System.Xml.XDocument" version="4.0.11-rc3-24208-04" />
+        <dependency id="Microsoft.DiaSymReader" version="1.0.8-rc2-60610-01" />
```

Comment:
The nuspec files don't contain version numbers anymore. Have you synced to the latest?